apps/system/js/notifications.js

```diff
@@ -624,18 +657,18 @@ var NotificationScreen = {
     notification.style.transform = '';
   },
 
-  addUnreadNotification: function ns_addUnreadNotification(id, skipUpdate) {
+  addUnreadNotification: function ns_addUnreadNotification(appManifestURL, skipUpdate) {
```

Comment:
actually you don't need to change signature of this function, you are using value of ``appManifestURL`` as an identifier for notifications counter.